pkg/systemtemplate/import.go

```diff
@@ -103,7 +102,8 @@ func SystemTemplate(resp io.Writer, agentImage, authImage, namespace, token, url
 		tolerations = templates.ToYAML(taints)
 	}
 
-	if cluster != nil && len(cluster.Spec.AgentEnvVars) > 0 {
+	if cluster != nil {
+		settings.MergeSettingsWithAgentEnvVars(cluster.Spec.AgentEnvVars)
```

Comment:
Is it possible for none of these settings to have values? If that the case we'll end up with AgentEnvVars being YAML representation of empty array vs. previously it being empty (period). Are we OK with this behavior change?